CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` when updating to this version!
 
+- Breaking: Changed the minimal supported Python version from 3.8 to 3.9. (#2397)
+- Minor: Add native support for the pyenv for managing Python versions. (#2397)
```

Comment:
I'm not sure, but this seems to me like it would sound better:
```suggestion
- Minor: Add native support for pyenv for managing Python versions. (#2397)
```